menu/drivers/xmb.c

```diff
@@ -2,7 +2,6 @@
  *  Copyright (C) 2011-2017 - Daniel De Matteis
  *  Copyright (C) 2014-2017 - Jean-André Santoni
  *  Copyright (C) 2016-2017 - Brad Parker
- *  Copyright (C) 2018 - Alfredo Monclús
```

Comment:
I would understand this if you changed to everything "The Libretro Team" or something like that, but this is telling me to piss off, since this is not a team, i will gladly go away.

Reply:
lol, likely was a git conflict he ran into. Unless you two have some beef to settle. :sunglasses:

Reply:
I don't think so, i came here to suggest that this shouldn't reposition left thumbnails to the right which is misleading, it should be a separate option, i even prepared a system to pass letter of position, expecting something like this happening, but i lost my temper when i saw this, only wanted something to have something that says "I helped", but seems like 1 line of acknowledgement is too much to ask

Reply:
I already told you before why i want no more credits in the menu code, and we told you specifically to take all the credits out of your PR before we would merge it. Go look through the comments, nobody was misled there except for us when it appeared this was still in xmb.c.

And now you want to make a huge controversy out of this when both Tatsuya79 and me are well aware of the conversation we had with you? 

The menu code will likely lead to a menu API that will be liberally licensed, we want zero individual copyright on it in the near future. I already communicated this to you before, go read the past thread. I have zero time or willingness to have some big fight over this, and this kind of immaturity generally is all the more reason why we should be more than careful not to have any individual copyright on files that have to be relicensed in the near future anyway.

Reply:
You not getting individual copyright in the menu code is not to spite you, it is like explained earlier, the entire system has to be turned into an API later on, it will need relicensing and the fewer people we have to ask permission to when it comes to that relicensing, the better and faster we can move.

That is it, really. It might be the case that XMB ultimately doesnt need to be relicensed and just some other parts of the current menu code do. Point is, we DO NOT KNOW YET, so I am just taking general precautions.

Reply:
If it turns out after the menu API/refactor work is done that XMB did not need to become a part of it and therefore need to be relicensed, you can get a copyright on xmb.c. That is the only way I can meet you halfway here.

Reply:
Beyond your copyright, to be very frank with you, if this attitude and the kind of drama you bring to the team is indicative of your future interactions with us, then perhaps this is for the best to be completely frank. Because one thing I do not want anymore is yet more stupid childish drama, it is toxic and it affects the entire team, and we have to waste time on dealing with temper tantrums like this one.

Reply:
If you're not allowing people to add their names, then just don't merge their contributions. As simple as that. If they add something relevant to the codebase they have the right to have their name there.

The copyright header doesn't really change the copyright anyway,  and as far as I can't tell relicensing that file would be a PITA, it has 43 contributors already.

Also while menu common code would be nice to have as MIT or whatever, xmb.c probably shouldn't be a part of that relicensing, what should be relicensed is a menu API so people can implement plug-able menus into retroarch without requiring the menu to be GPL too.

Reply:
I'm sorry, I just thought it was a leftover because of the discussion you had to not add copyright in that file.
It's not to disrespect you, and I don't even have my name there.

For the left part that goes bottom-right if you enable that option, I explained it in the sub-label.
That can be changed too if you have a better idea ready for that.

I know people would like even more, like 3 thumbnails where each one would be it's own entity/name/number. 
But the work here could be used as a base for scaling and auto-focus a picture if all thumbnails don't have something to display.

Reply:
> Also while menu common code would be nice to have as MIT or whatever, xmb.c probably shouldn't be a part of that relicensing, what should be relicensed is a menu API so people can implement plug-able menus into retroarch without requiring the menu to be GPL too.

That is the intent here all along. 

And I agree that xmbc.c probably isn't going to be part of that menu API.

In that light, @Tatsuya79 , reinstate @alfrix's copyright on xbmc.c.  

Beyond that, @alfrix, I'd appreciate if in the future you no longer start immediately acting abrasive towards us (and me in particular). My position on that still stands. That is the only issue I have still moving forward as far as our interactions go. This hostile relationship has got to stop, and we cannot have this. I refer to earlier comments you made too where I had to really bite my tongue and look the other way instead of resorting to taking the bait. If you agree to assuming good intent and we can avoid this, you have no quarrel with me.